woocommerce/woocommerce-android
# Feature/add metadata sources

This PR adds the source files for the Play Store metadata and a Fastlane lane to automatically update the related .pot files.

To Test:

1. Run `bundle exec fastlane update_ps_strings version:1.0` and verify that nothing changes in the .pot file.
2. Check out a new branch from this one.
3. Make some changes to one of the .txt files and commit them.
4. Run `bundle exec fastlane update_ps_strings version:1.0` and verify that the .pot file has been correctly updated. (Note that the lane may fail on the last git push command if the branch you are using doesn't have an upstream set).

Update release notes:
